### ISSUE TITLE
iOS app not recognizing Yubikey via NFC.

### ISSUE THREAD

**Issue by harrisondesbrosses** [standardnotes/forum#3446]
Hi, upon setting up a hardware key (Yubikey 5C NFC) on the web app, I cannot authenticate via NFC on iOS.

1. Login on iOS.
2. Click 'Authenticate'
3. Hold Yubikey 5C NFC to the back of the iPhone
4. Experience a time out, cancel out of the NFC dialogue and receive the message, "key...
{"name":"NotAllowedError","code":"ERROR_PASSTHROUGH_SEE_CAUSE_PROPERTY"}

I expect NFC to function properly within SN but it does not. I even see a Yubico Authenticator banner pop up while trying to get SN to notice the device.

 - OS: iOS 17.2.1
 - Version 3.183.39 (1007)

All other apps on my phone recognize the key and I'm not experiencing any other issues with the Yubikey/NFC. I have ordered a USB-C to lightning adapter in hopes that I can authenticate via lightning in place of NFC.

**Comment by dwin** [comment]
Did you try with the Yubico app installed? It operates as an extension in some contexts.

**Comment by harrisondesbrosses** [comment]
> Did you try with the Yubico app installed? It operates as an extension in some contexts.

Yes. Sorry if it was unclear but this is what I was referring to when I said, "I even see a Yubico Authenticator banner pop up while trying to get SN to notice the device." The banner could not be issued without the Yubico app installed. I'm still experiencing this issue.

Additionally, the USB adapters did not work. As of now I'm unable to use Standard Notes on my mobile.

**Comment by harrisondesbrosses** [comment]
I just tried again after uninstalling the Yubico iOS app. Nothing changed. I'm getting a bit desperate as I'm fully losing out on the ability to use SN on my iOS device and cannot drop hardware keys out of my workflow.

**Comment by senz** [comment]
I have similar error using FIDO2 key nitrokey 3: ERROR_PASSTHROUGH_SEE_CAUSE_PROPERTY
Im not sure why my login session was closed also, its either because of app update or some token expire. But it coincide with apps update and reboot, so I would bet on app degradation.

**Comment by harrisondesbrosses** [comment]
Any update? I had to drop Yubico keys from my SN workflow because of this issue and would love to implement them.

**Comment by belargam** [comment]
Getting the same issue. Yubikeys don't work via NFC, which means USB-C + NFC Yubikeys don't work. I ended up deleting the Yubikeys from the security settings for Standard Notes, and using the Yubico Authenticator instead. Quite a bit clumsier than just tapping the key, but it serves the purpose.
